SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Eiffel-community Vulnerability and Security Reporting and Response
+
+## Report a Vulnerability or Security Issue
+We as a community encourage researchers, users and contributors to report vulnerabilities and security related issues to the Eiffel community. All issues are thoroughly investigated by a community security officer and/or other community security volunteers. All reported and fixed security and vulnerability issues can be found on the [Eiffel community security page](https://eiffel-community.github.io/security.html) .
+
+## How to Report a Security Vulnerability
+To file a vulnerability report please send and e-mail to the private eiffel-community-security@googlegroups.com  list. The e-mail should list the security specific details as well as the [standard bug report information](https://github.com/eiffel-community/.github/blob/master/.github/ISSUE_TEMPLATE.md). Only the community security officers will have access to e-mails sent on the security and vulnerability list. This process is the same whether the report stems form an project within the Eiffel community or an external contributor. 
```

Comment:
Please rephrase to 'from a project within the Eiffel community or from an external contributor.'

Reply:
Done